### ISSUE TITLE
Link color with default template

### ISSUE THREAD

**Issue by ventoux84** [pmoreno-rodriguez/grav-theme-mundana#19]
In a default template page, color link is always green "success".

Changing the color choice in “page color style” does nothing.

**Comment by pmoreno-rodriguez** [comment]
I'm not sure what you mean. I've tried changing the page style on a default page, and everything works as it should. The color style for the page doesn't change the color of the links, but elements like buttons, the back to top button, social media icons, links to categories or tags, but not the normal links within the page.

**Comment by ventoux84** [comment]
The color of normal links within the page moved from blue to green. This has happened with the new version of Mundana; even though I haven't made any changes.

Do you have any idea why this might have happened?

**Comment by pmoreno-rodriguez** [comment]
Ok, now I understand.
In the latest version I made some adjustments to the css styles to make them the same as the original theme (https://wowthemesnet.github.io/mundana-theme-jekyll). However, I have been able to see that the hover attribute color on links is still blue.

I don't know yet what decision to make, but I could change the normal links to the style of the page as well, as well as the buttons and other elements that change color.
